src/deploy.cmd.js

```diff
@@ -331,7 +337,6 @@ class DeployCommand extends AbstractCommand {
 
       const triggeroptions = {
         method: 'POST',
-        json: true,
         auth,
```

Comment:
Note that this is a `request` specific option. It was never supported by `helix-fetch` (nor by `node-fetch`) ... 

This code never worked.